hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/metadata/SparkHoodieBackedTableMetadataWriter.java

```diff
@@ -162,8 +170,10 @@ protected void commit(String instantTime, Map<MetadataPartitionType, HoodieData<
         // clean plan is the same, so we don't need to delete the requested and inflight instant
         // files in the active timeline.
       }
-      
-      List<WriteStatus> statuses = writeClient.upsertPreppedRecords(preppedRecordRDD, instantTime).collect();
+
+      List<WriteStatus> statuses = partitioner.isPresent()
```

Comment:
we are using partitioner to deduce whether its first commit or subsequent one. 
wondering if we can add a new commitMethod altogether only for initial commit and keep it light weight (we don't need to trigger compaction, cleaning etc) and directly do bulk_insert for eg.